src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
-
-console.log('hello mate academy!');
+const clickShow = document.querySelector('#click-text');
+let clickNumber = sessionStorage.getItem('clickNumber') || 0;
+clickShow.innerText = `You have clicked ${clickNumber} times`;
+function increaseClick() {
+  clickNumber++;
+  sessionStorage.setItem('clickNumber', clickNumber);
+  clickShow.innerText = `You have clicked `
+    + `${sessionStorage.getItem('clickNumber')} times`;
```

Comment:
ти можеш використати `clickNumber`
```suggestion
    + `${clickNumber} times`;
```